frontend/src/views/Collection/components/ActionButtons/style.ts

```diff
@@ -0,0 +1,10 @@
+import { PT_GRID_SIZE_PX } from "src/components/common/theme";
+import styled from "styled-components";
+
+export const StyledDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  & > :not(:last-child) {
```

Comment:
Just curious - what is this line doing?

Reply:
Great question! 

This is saying that for any direct child element that is not the last child, we apply the styles 🤩👍